### ISSUE TITLE
FieldError on "Save for all" text word translations

### ISSUE THREAD

**Issue by bgins** [ereadingtool/ereadingtool#253]
When attempting to "Save for all" in the translations tab of the text editor, the backend fails with:

```
FieldError at /api/text/translations/match
Cannot resolve keyword 'word_type' into field. Choices are: aspect, form, id, instance, instructor_flashcards, 
mood, phrase, pos, student_flashcards, tense, text_section, text_section_id, textword, textwordgroup, 
translations
```

To reproduce, open a pre-existing text that contains repeated instances of a word. In the translations tab, select the word and add a translation or edit a grammeme. The "Save for all" button should appear. Click it and the watch the backend for the error message.

**Comment by jdelamare** [comment]
Hey there, I rolled back to a commit on `upgrade/staging` on November 11th. I still can't seem to find the "Save for all" button, but it could be there in plain sight. Would you help point it out to me please? Then I can address this error.

**Comment by bgins** [comment]
"Save for all" should only save for words that are currently selected for editing. So the screenshot above would not show the "Save for all" button, unless the second instance of the word was also selected.

**Comment by bgins** [comment]
Fixed in #351
